<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt64</class>
 <widget class="QMainWindow" name="Qt64">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Qt64</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="loMain" stretch="2,4">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="statusTip">
         <string>Double-click to run</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="lovRightGroup" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnRun">
            <property name="statusTip">
             <string>Start the selected image</string>
            </property>
            <property name="text">
             <string>Boot cartridge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPatch">
            <property name="statusTip">
             <string>Upload changes from buffer to cartridge</string>
            </property>
            <property name="text">
             <string>Patch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnErase">
            <property name="statusTip">
             <string>Eras buffer and cartridge contents</string>
            </property>
            <property name="text">
             <string>Erase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnReset">
            <property name="statusTip">
             <string>Reset the Commodore 64</string>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Information</string>
             </attribute>
             <layout class="QVBoxLayout" name="presentation">
              <item>
               <widget class="QLabel" name="gameTitle">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>22</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Game Title</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gameScreen">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Cartridge editor</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPrewferences">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_ROM_directory"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuPrewferences"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_ROM_directory">
   <property name="text">
    <string>Set ROM directory</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnRun</sender>
   <signal>clicked()</signal>
   <receiver>Qt64</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPatch</sender>
   <signal>clicked()</signal>
   <receiver>Qt64</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Qt64</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnErase</sender>
   <signal>clicked()</signal>
   <receiver>Qt64</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
